lib/cocoapods/resolver/resolver_specification.rb

```diff
@@ -17,10 +17,28 @@ class ResolverSpecification
       attr_reader :used_by_non_library_targets_only
       alias used_by_non_library_targets_only? used_by_non_library_targets_only
 
-      def initialize(spec, used_by_non_library_targets_only, source)
+      # @return [Bool] whether this resolved specification is a transitive dependency that was not directly included
+      #         by the target that depends on it
+      #
+      attr_reader :transitive
+      alias :transitive? transitive
+
+      # @return [Array<Specification>] specifications which depend on this specification
+      #
+      attr_reader :incoming_vertices
```

Comment:
could use some suggestions for a better name

Reply:
`dependent_specs`?

Reply:
yes. nice